src/components/connect.js

```diff
@@ -330,6 +341,7 @@ function connect(mapPropsToRequestsToProps, defaults, options) {
                 [prop]: mapping
               }),
             data: Object.assign(
+              {},
```

Comment:
should I make this change to the other state objects as well? I don't need to because I never compare them like I do `data`, but maybe it'd be more consistent?